### ISSUE TITLE
The zoom function is regularly frozen on iMac

### ISSUE THREAD

**Issue by pcmusic75** [photopea/photopea#7204]
Hi,

Since a couple of weeks, Photopea is becoming nearly unusable because the image zoom function is constantly frozen on iMac.

When I want to zoom to work on an image, I'm naturally using Alt+ scroll up and down with my mouse. Sometimes I have to reload the image 5-6 times to make it work; Sometimes I have to restart my Firefox browser... but it doesn't work very often.
Before I never met that kind of issue.

Thanks to fix this major bug.

Cheers.

**Comment by photopea** [comment]
Coult you tell us what sould I do to see it myself? I have a macbook air M1 with the latest MacOS.

**Comment by photopea** [comment]
I have to close it because of lack of details :(

**Comment by pcmusic75** [comment]
Hi,
You open any image from your laptop to do what you want with it and zoom it to see more details.
I'm using alt+magic mouse to zoom in. This function is nearly frozen and the image remains blocked with a small size. Sometimes it unblocks.
I tried to use Photopea with Chrome and I don't have this issue.
I guess the frozen image might be a compatibility problem related with Firefox latest update.
Thanks for your reply.

**Comment by photopea** [comment]
Please, make a proper report and fill out the report template. I do not have a magic mouse, so please, describe how to make your problem happen without it.

**Comment by pcmusic75** [comment]
Sorry, I'm not a geek at all. I'm only using basic functions of Photopea.
When I open an image with Photopea in Firefox, many times it can't zoom in or out by using the keyboard shortcuts. It stays frozen at the same size and I can't work on the image because it's too small or I can't change its size when necessary.
I don't have this issue when I using Photopea with Chrome.
Is there a way to reset Photopea to default setting? 
Could it be a Firefox problem?
Is there compatibility problem with latest updates of Photopea and Firefox?
Thanks for your reply.

**Comment by photopea** [comment]
Please, make a proper report and fill out the report template. Can you see the report template? Just answer the questions there.

**Comment by pcmusic75** [comment]
**Can you see the report template?**
Sorry no! Where can I find it?

**Comment by photopea** [comment]
When you press "New Issue" on Github, there is "Bug Report", you should press "Get Started".
